cg/server/api.py

```diff
@@ -187,16 +188,14 @@ def family(family_id):
     return jsonify(**data)
 
 
-@BLUEPRINT.route("/families_in_customer_group/<family_id>")
-def family_in_customer_group(family_id):
+@BLUEPRINT.route("/families_in_collaboration/<family_id>")
+def family_in_collaboration(family_id):
```

Comment:
When is this route used? Can be removed?

Reply:
It is used by cgweb